index.js

```diff
@@ -219,6 +223,8 @@ class Raspi extends EventEmitter {
           }
           if (pinInfo.peripherals.indexOf('pwm') != -1) {
             supportedModes.push(PWM_MODE, SERVO_MODE);
+          } else if (enableSoftPwm === true && pinInfo.peripherals.indexOf('gpio') != -1) {
```

Comment:
style nit: let's normalize this and use strict equality checking for both parts of the conditional, not just the first, aka:

```JavaScript
} else if (enableSoftPwm === true && pinInfo.peripherals.indexOf('gpio') !== -1) {
```